populate/src/main/resources/seed/portals/ourhealth/studies/ourheart/study.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "OurHealth",
   "shortcode": "ourheart",
+  "pepperStudyName": "juniper-ourhealth",
```

Comment:
this should use the study shortcode -- rather than the portal shortcode.  I wonder if the right move is to have the pepper study name always be `juniper-${studyShortcode}`.  That way we don't need an extra DB field, DSM gets an easy distinction, and we can hardcode the prefix to ensure consistency

Reply:
That's a perfect middle-ground. We're going with `juniper-${studyShortcode}`.